mail/tasks.py

```diff
@@ -0,0 +1,97 @@
+"""Tasks for mail app"""
+import logging
+from urllib.parse import urljoin
+
+from django.conf import settings
+from django.urls.base import reverse
+from celery import shared_task
+
+from mail import api
+from mail.models import NotificationEmail
+from ui.constants import VideoStatus
+
+
+log = logging.getLogger(__name__)
+
+
+def _get_recipients_for_video(video):
+    """
+    Returns a list of recipients that will receive notifications for a video.
+
+    NOTE: this needs to be updated to include all the admin moira lists associated to the collection
+    where the video is.
+
+    Args:
+        video (ui.models.Video): a video object
+
+    Returns:
+        list: a list of strings representing emails
+    """
+    recipients_list = []
+    owner_email = video.collection.owner.email
+    if owner_email:
+        recipients_list.append(owner_email)
+    return recipients_list
+
+
+STATUS_TO_NOTIFICATION = {
+    VideoStatus.COMPLETE: NotificationEmail.SUCCESS,
+    VideoStatus.TRANSCODE_FAILED_INTERNAL: NotificationEmail.OTHER_ERROR,
+    VideoStatus.TRANSCODE_FAILED_VIDEO: NotificationEmail.INVALID_INPUT_ERROR,
+    VideoStatus.UPLOAD_FAILED: NotificationEmail.OTHER_ERROR,
+}
+
+
+def send_notification_email(video):
+    """
+    Sends notification emails to the admin of the channels where a video belongs
+    according to the video status.
+
+    Args:
+        video (ui.models.Video): a video object
+    """
+    if video.status not in STATUS_TO_NOTIFICATION.keys():
+        log.error("Tried to send notifications for video %s with status %s", video.hexkey, video.status)
+        return
+    # get the list of emails
+    recipients = _get_recipients_for_video(video)
+    if not recipients:
+        log.error("No email was sent because there were no "
+                  "valid recipient emails for video %s with status %s", video.hexkey, video.status)
+        return
+    try:
+        email_template = NotificationEmail.objects.get(notification_type=STATUS_TO_NOTIFICATION[video.status])
+    except NotificationEmail.DoesNotExist:
+        log.error("No template found for error %s", STATUS_TO_NOTIFICATION[video.status])
+
+    try:
+        api.MailgunClient.send_individual_email(
```

Comment:
`MailgunClient.send_individual_email` uses `MailgunClient. send_batch` .

I am `MailgunClient.send_individual_email` because in theory we need to send just one email.